Exercise_D_E/assets/style/index.css

```diff
@@ -2,3 +2,72 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+body{
+    padding: 0px;
+    margin: 0px;
+    font-family:'Roboto', sans-serif;
+}
+
+.logo{
+    padding-left: 30px;
+}
+
+.backgroundimg{
+    width: 100%;
+    filter: brightness(50%);
+}
+
+.container1{
+    color: white;
+    padding: 0px;
+    margin: 0px;
+}
+
+.center_text{
+    font-size: 50px;
+    position: absolute;
+    top: 60%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.button{
+    font-size: 30px;
+    position: absolute;
+    top: 80%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background-color: rgb(30, 229, 30);
+    padding: 10px 20px;
+    border-radius: 16px;
+}
+
+.container2, .container3{
+    color: grey;
+    background-color: rgb(200, 238, 200);
+    font-size: 20px;
+    display: flex;
+    justify-content: baseline;
+    flex-wrap: wrap;
+    position: relative;
+}
+
+.box{
+    width: 200px;
+    height: auto;
+    padding: 100px 30px;
+}
+
+.icon{
+    width: 80px;
+    height: auto;
+}
+
+.text{
+    text-align: center;
+}
+
+.photo{
+    text-align: right;
+}
```

Comment:
I would suggest for container2 & container3, you can use `    color: grey;
    background-color: rgb(200, 238, 200);
    font-size: 20px;
    display: flex;
    justify-content: space-around;
    flex-wrap: wrap;
    position: relative;
    flex-direction: row;
    align-content: space-around;
    align-items: stretch;`

Reply:
For the Flex, you can use Justify-content:space-around and also align-items:stretch